coalahtml/coala_html.py

```diff
@@ -35,7 +35,7 @@ def main():
 
     dir_path = create_dir(os.path.abspath(args.dir))
 
-    if len(list(os.walk(dir_path))) > 0:
+    if len(list(os.walk(dir_path))) > 0:  # pragma: no cover
```

Comment:
>Just check if `dir_path/index.html` exists.
This feels like a little too hardcoded. If you want to check if you didn't copy it already, why not check exactly that, that every file from the path you're copying is here already. This way if you're for example updating coala-html, e. g. adding new files, user will get those new files copied. With your approach, he will have to manually remove this directory.